README.md

```diff
@@ -7,5 +7,7 @@ The Zbooni echo system user guide.
   - [Splash Screen](apps/iOS/splash-screen.md)
   - [Create Account](apps/iOS/create-account.md)
   - [Account Login](apps/iOS/account-login.md)
+  - [Navigation Bar](apps/iOS/navigation-bar.md)
+  - [New Order](apps/iOS/new-order.md)
```

Comment:
@omnia-95 Better use the term `Create Order` here.

Reply:
I was thinking to name it according to the navigation bar labels, cause you can also create a new order in 'orders' screen if you haven't received any orders yet, also in the navigation-bar.md you mentioned writing the exact labels,  we can have it like you did with splash screen where its blue text that redirects you to the screen